### DISCUSSION TITLE
HR CAM7 simulations - version 2

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#587] [General]
Discussion #585 has become quite messy, so I am locking and start a new discussion summarizing where we stand.

-----

We are planning a  series of CAM7 simulations varying both horizontal and vertical resolutions. Specifically, we will perform AMIP simulations with observed sea surface temperature (SST) data and SST+4K at 1° and 0.25° resolutions, with both the CAM6 and CAM7 grids (i.e, with 32, 58, 93 vertical levels). 

These simulations will be used to investigate the impact of both vertical and horizontal resolution on climate sensitivity, regional precipitation patterns, gravity waves, boundary layer aerosol-cloud interactions and hurricane activity

-----

## Configuration
**- Run type: AMIP type for years 2000-2010**
**- Tag: cam6_4_036**
**- Output: We collected the request for output [here](https://docs.google.com/document/d/1T9T8Axz1L58G0YB__QNXsx5u4rpUHEhXdFFcVQP6uJY/edit)**
Output will be found:
```
/glade/campaign/cesm/community/amwg/cam_hr
```

-----

## List of runs

10 yr = 2000-2010
5yr = 2000-2005
| Physics      | Res       | Levels | Experiment | Period| Comments | Issue|
|--------------|-----------|--------|------------|:-----:|:--------:|:----:|
| cam7         | 1 deg     | L32    | SST        | 10 yr |**Done**|#596|                                
|              |           |        | SST+4K     |  5 yr |**Done**|#597|                                 
|              |           | L58    | SST        | 10 yr |**Done**|#598|                   
|              |           |        | SST+4K     |  5 yr |**Done**|#599|
|              |           | L93    | SST        | 10 yr |**Done**|#600|
|              |           |        | SST+4K     |  5 yr |**Done**|#601|
|              | 0.25 deg  | L32    | SST        | 10 yr |**Done**|#593|
|              |           |        | SST+4K     |  5 yr |**Done**|#595|
|              |           | L58    | SST        | 10 yr |**Done**|#591|
|              |           | L93    | SST        | 10 yr |**Done**|#590|  
|              |           |        | SST+4K     |  5 yr |**Done**|#594|
|  |   |   |  |  | |  | |
| prog mom off | 0.25 deg  | L93    | SST        |  5 yr |**Done**|#592|
|  |   |   |  |  | |  |
| cam5         | 1 deg     | L32    | SST        |  5 yr |checking cam5|#603|
|              |           | L58    | SST        |  5 yr | tuning                   | |
|              |           | L93    | SST        |  5 yr | does cam5 L93 work?      | |
|              | 0.25 deg  | L58    | SST        |  5 yr | 5 or 10 yrs ? L58 or L93?  |   |
|              |           |        | SST+4K     |  5 yr   |         | |

@JulioTBacmeister, @adamrher, @islasimpson, @swrneale, @PeterHjortLauritzen, @cmcclus,  @bstephens82, @megandevlan, @jedwards4b, @billsack, @wwieder, @ekluzek, @shieldsca, @slevis-lmwg,

**Comment by cecilehannay** [comment]
## Estimate of the cost and output size

_____

### fincl 
The variable and fincl are discussed here:
https://docs.google.com/document/d/1T9T8Axz1L58G0YB__QNXsx5u4rpUHEhXdFFcVQP6uJY/edit
_____

### Cost estimate
- Model Cost:          305201.25   pe-hrs/simulated_year
- Model Throughput:    0.48   simulated_years/day

_____

### Output estimate per month

#### On interpolated ne120 grid
- monthly - average on native grid for ADF    => **2.7G**
- daily - average on native grid for MDTF     => **1.2G** 

#### On native grid
- monthly - average => **21G**
  - our usual monthly output (15G)
  - short list from Christina (6.6G)
- monthly - average => **556K**
  - the usual zm variables (does this work?)
- daily - average  => **1.4 G**  
  - our usual daily average 
- 6 hourly - instantaneous => **374G** 
  - Julio’s list (202G) 
  - short list from Ben (172G) 
- 3 hourly - average => **1.5G** 
  - Precip  
- 3 hourly instantaneous  => **15G**
  - TC and AR request (from Ben, Julio Christine)

### Output estimate  per year:
- without the 6 hourly instantaneous: 43.3G*12 = **520G per year** 
- plus the 6 hourly instantaneous: 374G*12 = **4368G per year**
- Land = 15G*12 = **180G per year**
- **TOTAL = about 5T per year**

**Reply by cecilehannay** [reply]
We are ready to start the simulation. 

- I did a **cost estimate**
```
Model Cost: 305201.25 pe-hrs/simulated_year
Model Throughput: 0.48 simulated_years/day
```
- and an **output estimate**
```
about 5T per year
```
____________

**Please check the output to ensure that the variables you need are being saved and at the correct frequency.**.

You can either:
- review the `fincl` in the googledoc [here](https://docs.google.com/document/d/1T9T8Axz1L58G0YB__QNXsx5u4rpUHEhXdFFcVQP6uJY/edit?usp=sharing)
- or directly loolk at the output on glade:
```
/glade/derecho/scratch/hannay/archive/f.e30_cam6_4_036.FMTHIST.ne120_L93.001_t4
```

**Comment by cecilehannay** [comment]
# Land Setting

@adamrher created land files that I will use for this run:
```
flanduse_timeseries = '/glade/derecho/scratch/aherring/restart/ctsm5.2.028/tools/mksurfdata_esmf/landuse.timeseries_ne120pg3_SSP3-7.0_1979-2026_78pfts_c240925.nc' 
fsurdat = '/glade/derecho/scratch/aherring/restart/ctsm5.2.028/tools/mksurfdata_esmf/surfdata_ne120pg3_SSP3-7.0_1979_78pfts_c240925.nc'
```

**Reply by slevis-lmwg** [reply]
@cecilehannay @adamrher 
I apologize if this question is already resolved and closed, but I wanted to make sure:
@ekluzek should Adam have generated these files with ctsm5.3.0 to include our corrections to soil properties and Bill's changes to glaciers (etc. etc.) OR...
- did Adam already accomplish this with temporary fixes?
- did we decide to continue with ctsm5.2 datasets for now?

**Reply by adamrher** [reply]
Since the ne120pg3 runs are imminent, we were going to run with ctsm5.2. I implemented the soil texture bug fix (and double checked ORGC in the file above with with ncview), but I was not aware there are glacier changes slated for ctsm5.3. What are those?

**Reply by slevis-lmwg** [reply]
Copying from ctsm5.3.0 ChangeLog:
```
With new surface datasets the following GLC fields have region "1" set to UNSET:
     glacier_region_behavior, glacier_region_melt_behavior, glacier_region_ice_runoff_behavior
[...]
   Region 1 for glacier region is now undefined on purpose
```

**Reply by adamrher** [reply]
We certainly want the downscaled mass balance calculated by CLM -- does this still work in ctsm5.2, @billsacks?

**Reply by billsacks** [reply]
Yes - you'll still get downscaled mass balance. The difference is that, previously, you would get SMB extending beyond Greenland itself, covering the full CISM domain (including, e.g., Ellesmere Island); now you'll only get SMB over Greenland itself. (There is some rationale in https://github.com/ESCOMP/CTSM/issues/423.)

**Reply by slevis-lmwg** [reply]
@adamrher I'm pretty sure that this changes answers, but the change is minor.

**Reply by slevis-lmwg** [reply]
Though maybe I'm thinking in land-only context...

**Reply by adamrher** [reply]
I have no problem with the extended area, I usually do my own masking in post-processing anyway. And since this is a FLTHIST run, we don't have to worry about the negative runoff fluxes and the related corrections, because they don't occur, right?

**Reply by billsacks** [reply]
> @adamrher I'm pretty sure that this changes answers, but the change is minor.

Yes, the glacier region change does change answers.

> since this is a FLTHIST run, we don't have to worry about the negative runoff fluxes and the related corrections, because they don't occur, right?

There could still be negative runoff fluxes, but they shouldn't matter unless you're explicitly looking at negative runoff fluxes.

**Reply by ekluzek** [reply]
I just did a quick look, and @adamrher used ctsm5.2.028 with the fix to soil data to create those datasets that you can use in the ctsm5.2 tag you are using. So that all looks good to me.

**Reply by cecilehannay** [reply]
for the ne30 equivalent, @adamrher created a fsurdat to fix soil data. 
```
'/glade/derecho/scratch/aherring/restart/ctsm5.2.028/tools/mksurfdata_esmf/surfdata_ne30pg3_SSP3-7.0_1979_78pfts_c241014.nc'
'/glade/derecho/scratch/aherring/restart/ctsm5.2.028/tools/mksurfdata_esmf/landuse.timeseries_ne30pg3_SSP3-7.0_1979-2026_78pfts_c241014.nc'

```

As a result, the ne30 runs use the following datasets:
```
finidat = '/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/initdata_esmf/ctsm5.3/ctsm52018_f09_xsiceON_acON_pSASU.clm2.r.0201-01-01-00000.nc'
fsurdat = '/glade/derecho/scratch/aherring/restart/ctsm5.2.028/tools/mksurfdata_esmf/surfdata_ne30pg3_SSP3-7.0_1979_78pfts_c241014.nc'
flanduse_timeseries = '/glade/derecho/scratch/aherring/restart/ctsm5.2.028/tools/mksurfdata_esmf/landuse.timeseries_ne30pg3_SSP3-7.0_1979-2026_78pfts_c241014.nc'

```

**Reply by slevis-lmwg** [reply]
I want to point out that @olyson has generated a new spun up finidat file for ne30, specific to 1979:
`/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/initdata_esmf/ctsm5.3/ctsm53n04ctsm52028_ne30pg3t232_BgcCrop_exice_hist.clm60.r.1979-01-01.nc`
This and other new finidat files are slated to come in with the beta04 tag.

**Reply by cecilehannay** [reply]
Thanks, @slevis-lmwg. Do you recommend I use that file? My run starts in 2000 but I assume it should be good enough.

**Reply by slevis-lmwg** [reply]
In the same directory we have a new 2000 file
`ctsm53n04ctsm52028_ne30pg3t232_BgcCrop_exice_hist.clm60.r.2000-01-01.nc`

**Reply by slevis-lmwg** [reply]
I suggest that you try it. If for some reason it causes trouble, go back to your original plan for now.

**Comment by JulioTBacmeister** [comment]
# List of runs
Don't think we need L93 CAM7 SST+4K for 10 years
Don't need prognostic mom flux OFF SST+4K
Would like 1 year of CAM5 L93 ... sometime.

General comment: Comparing L93 CAM7 to L58 CAM5 should be OK for examing how high-res impacts tropospheric climate (@islasimpson ?)

**Reply by cecilehannay** [reply]
I revised the above table `List of Runs` following discussion with @islasimpson and @JulioTBacmeister. 
- We added cam5/1deg experiments to tune cam5 before moving to ne120.  
- We will check whether we can do cam5 at L93. 
- We might want to drop all L58 experiments and only do L93 ?

**Comment by swrneale** [comment]
# SSTs

While I understand the +4K SST experiments and how they can reveal climate sensitivities, the patterns of observed SST anomalies are much more complex and subtle that this. I'm wondering whether there's space here to do something a bit more nuanced such as maybe adding 2x or 3x the observed spatial distribution of SST increases from pre-industrial to present day? The pattern of recent SST change is quite detailed and the structure of change in the N. Atlantic. likely affects tropical cyclones and the stratocumulus regions have experienced SST change of opposite sign. Just a thought.  ![image](https://github.com/user-attachments/assets/81ab3fce-bbaf-4e4a-9d33-eb6705063886)

**Reply by cecilehannay** [reply]
Another unrelated comment about SSTs: 
The 2000-2010 runs are using the HadSST 1 degree:
```
/glade/campaign/cesm/cesmdata/cseg/inputdata/atm/cam/sst/sst_HadOIBl_bc_1x1_1850_2021_c120422.nc
```
I wanted to make sure we are happy with using that dataset.

**Reply by swrneale** [reply]
Are you saying we need a higher resolution dataset or a different dataset source? Do we want daily as well? sst_reynolds_daily_0.9x1.25_yrs1982-2018_timeseries_c20190822.nc
But it is time limited.

**Comment by cecilehannay** [comment]
All the runs completed (although we will extent `prog mom off` to 10 years). 
I am very close to my quota and I am moving the data to `campaign` in:
```
/glade/campaign/cesm/community/amwg/cam_hr
```
although it will take a few to complete the process on globus.
